README.md

```diff
@@ -142,7 +142,7 @@ To run the project, execute the following command:
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be great if you made your Key Features more meaningful to make your readme descriptive and professional 😊

Reply:
Absolutely! I will make the suggested changes right away 🎯 ! Cheers Hammas! 😃